GPU/Common/SoftwareTransformCommon.cpp

```diff
@@ -331,12 +324,8 @@ void SoftwareTransform(
 					}
 				}
 			} else {
-				if (reader.hasColor0()) {
-					for (int j = 0; j < 4; j++) {
-						c0[j] = unlitColor[j];
-					}
-				} else {
-					c0 = Vec4f::FromRGBA(gstate.getMaterialAmbientRGBA());
```

Comment:
This change mystified me a bit until I went to read the full code... Yeah, this does seem redundant, we already check hasColor0 earlier and initialize unlitColor accordingly. This is just a note to self in case this change needs to be revisited..

Reply:
Right, the commit comment has "In this case, unlitColor is already materialAmbient" too.

-[Unknown]

Reply:
Sorry, I missed that - I really should review commit by commit instead of the whole diff at once, which I sometimes do.

Reply:
Heh, either way bites you.  Just recently I was reviewing a pull and half the commits were reverted later in the pull (there was actually a good reason for it, but was still kicking myself.)

-[Unknown]